Habinezajanvier/my-brand-api
# Ch set eslint #9

#### What does this PR do?
Setting Eslint for linting
#### Description of Task to be completed?
- Set eslint for development linting
- Set HoundCI for PR code validations commenting
#### How should this be manually tested?
- Clone the repo `git clone`
- Switch the directory `cd my-brand-api`
- Checkout the branch `git checkout ch-set-eslint-#9`
- Start the app in VSCode
- Start editing some files
> You should see code violations comments and errors
#### Any background context you want to provide?
Eslint and prettier extension for VSCode should be installed
#### What are the relevant pivotal tracker stories?
https://trello.com/c/j0Tbz3QF/9-setting-up-eslint
